docs/src/pages/milestones.tsx

```diff
@@ -55,12 +55,20 @@ import {
   mdiChartBoxMultipleOutline,
   mdiAccountGroupOutline,
   mdiFlowerPoppy,
+  mdiHandshakeOutline,
 } from '@mdi/js';
 import Layout from '@theme/Layout';
 import React from 'react';
 import Timeline, { DateType, Item } from '../components/timeline';
 
 const items: Item[] = [
+  {
+    icon: mdiHandshakeOutline,
+    description: 'Joined Futo and Immich core team goes full-time',
+    title: 'Immich joins FUTO!',
+    date: new Date(2024, 4, 1),
```

Comment:
2024-04-01, or 2024-05-01?

Reply:
In javascript, months are 0-indexed, so 4-1 is correct and as you can see in the screenshot shows the correct date.

Reply:
ah, so silly of me. my bad 😅